src/mainwindow.cpp

```diff
@@ -6223,6 +6224,7 @@ void MainWindow::openLocalUrl(QString urlString) {
         Note note;
 
         if (isNoteFileUrl) {
+            ui->noteSubFolderTreeWidget->reset();
```

Comment:
@fnkabit, do you by change remember why you added this line? It causes #2128.

Reply:
@pbek Ah..sorry, don't remember.

Reply:
No worries, I removed the line for now. 😁